pkg/validator/handler.go

```diff
@@ -61,7 +63,7 @@ func Handler(validator Validator) http.HandlerFunc {
 	}
 }
 
-func writeResponse(validator Validator, writer http.ResponseWriter, response *v1beta1.AdmissionResponse) {
+func writeResponse(validator generic.Logger, writer http.ResponseWriter, response *v1beta1.AdmissionResponse) {
```

Comment:
Similar to `mutator`, maybe rename `validator` to `logger` or something along the lines :slightly_smiling_face: